Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Compiled Lua sources\r\nluac.out\r\n\r\n# luarocks build files\r\n*.src.rock\r\n*.zip\r\n*.tar.gz\r\n\r\n# Object files\r\n*.o\r\n*.os\r\n*.ko\r\n*.obj\r\n*.elf\r\n\r\n# Precompiled Headers\r\n*.gch\r\n*.pch\r\n\r\n# Libraries\r\n*.lib\r\n*.a\r\n*.la\r\n*.lo\r\n*.def\r\n*.exp\r\n\r\n# Shared objects (inc. Windows DLLs)\r\n*.dll\r\n*.so\r\n*.so.*\r\n*.dylib\r\n\r\n# Executables\r\n*.exe\r\n*.out\r\n*.app\r\n*.i*86\r\n*.x86_64\r\n*.hex\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 9ee3f4d7bca2053603abc8d6ca982b8c4dd130dc)
+++ b/.gitignore	(date 1714013332000)
@@ -38,4 +38,8 @@
 *.i*86
 *.x86_64
 *.hex
-
+*.apk
+*.msi
+*.deb
+*.rpm
+*.jar
